docs/content/preview/yugabyte-platform/manage-deployments/remove-nodes.md

```diff
@@ -16,6 +16,40 @@ menu:
 type: docs
 ---
 
+## Replace a live or unreachable node
+
+If a node (live or unreachable) needs to be replaced, do the following:
+
+1. Navigate to **Universes**, select your universe, and open the **Nodes** tab.
+1. Click the associated node **Actions > Replace Node**.
+
+    ![Replace Node Actions](/images/ee/replace-node.png)
+
+Clicking the **Replace Node** button opens a popup window to confirm the replacement of this node. Confirming the task redirects to the task page and will take a few minutes to complete.
+
+For Infrastructure as a service (IaaS) such as AWS and GCP, YugabyteDB Anywhere returns the node back to the IaaS. For on-premises universes, the unresponsive node will most likely have failed to clean up its data directory, home directory, and more. So it will be placed in a Decommissioned state in the on-premises provider which prevents the node from being added to a new universe.
+
+### Check on-premises node state
+
+The on-premises node instances have three states: In use, Free, and Decommissioned and the following illustration describes the workflow of the states.
+
+![Decommissioned node workflow](/images/ee/decommissioned-node.png)
```

Comment:
We can remove the "Action: Decommission", as we do not support this currently

Reply:
Also instead of release node in the "Release node: data cleanup fails" and "Release node:  cleanup succeeds" let's change "Release node" to "Replace node"